src/site/content/en/payments/setting-up-a-payment-method/index.md

```diff
@@ -0,0 +1,474 @@
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <code><a href="https://developer.mozilla.org/docs/Web/Manifest/name">name</a></code> (required)
+        </td>
+        <td>
+        Used as the payment app name.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code><a href="https://developer.mozilla.org/docs/Web/Manifest/icons">icons</a></code> (required)
+        </td>
+        <td>
+        Used as the payment app icon. Only Chrome uses these icons; other
+        browsers may use them as fallback icons if you don't specify them as
+        part of the payment instrument.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code><a href="https://developer.mozilla.org/docs/Web/Manifest/serviceworker">serviceworer</a></code>
+        </td>
+        <td>
+        Used to detect the service worker that runs as the web-based payment
+        app.
+        </td>
+      </tr>
+      <tr>
+        <td><code>serviceworker.src</code></td>
+        <td>
+        The URL to download the service worker script from.
+        </td>
+      </tr>
+      <tr>
+        <td><code>serviceworker.scope</code></td>
+        <td>
+        A string representing a URL that defines a service worker's
+        registration scope.
+        </td>
+      </tr>
+      <tr>
+        <td><code>serviceworker.use_cache</code></td>
+        <td>
+        The URL to download the service worker script from.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code><a href="https://developer.mozilla.org/docs/Web/Manifest/related_applications">related_applications</a></code>
+        </td>
+        <td>
+        Used to detect the native app that acts as the native payment app.
+        Find more details at <a href="/native-payment-apps-overview">Android
+        payment apps developer guide</a>.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code><a href="https://developer.mozilla.org/docs/Web/Manifest/prefer_related_applications">prefer_related_applications</a></code>
+        </td>
+        <td>
+        Used to determine which payment app to launch when both a native payment app and a web-based payment app are available.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code><a href="https://github.com/sahel-sh/shipping-contact-delegation/blob/master/Explainer.md#edit-just-in-time-jit-installation">payment.supported_delegations</a></code>
+        </td>
+        <td>
+        A string array used to determine the additional information that the
+        payment app can provide. `shippingAddress`, `payerName`, `payerEmail`,
+        and `payerPhone` are valid values.
+        </td>
+      </tr>
+    </tbody>
+    <caption>Important web app manifest fields</caption>
+  </table>
+</div>
+
+![](web-app-manifest.png)
+
+The web app manifest's `name` property is used as the payment app name, `icons`
+property is used as the payment app icon.
```

Comment:
* Needs an `alt`
* You can use the paragraph as a caption: https://web.dev/handbook/markup-media/#images-with-captions